Library/Formula/ucspi-tools.rb

```diff
@@ -0,0 +1,18 @@
+class UcspiTools < Formula
+  homepage "https://github.com/younix/ucspi/blob/master/README.md"
+  url "https://github.com/younix/ucspi/archive/v1.2.tar.gz"
+  sha1 "38a708efd6d72e0d9d077efb15477763bdea39b0"
+
+  depends_on "pkg-config" => :build
+  depends_on "ucspi-tcp"
+  depends_on "libressl"
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    out = shell_output("#{bin}/socks 2>&1", 1)
+    assert_equal "tcpclient PROXY-HOST PROXY-PORT socks HOST PORT PROGRAM [ARGS...]\n", out
```

Comment:
We need a socks-proxy for the socks-tool. Is it possible to ssh on localhost without password in the test framework?
In https://github.com/t6n/homebrew/commit/77943d622f2b05f8e09ba1b13d82008810b43d49 I tested the tlsc-tool. But the test framework says: "fatal: no IP address for github.com". We need any https (port 443) connection for the tlsc-tool. Which https service can we reach in the test framework?